### PR TITLE
Fix bug 1461199 - Translation length check is able to correctly count…

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz could you review?

### CODE REVIEW

File: pontoon/checks/libraries/pontoon.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
+import HTMLParser
 import re
 
+import bleach
+
 from collections import defaultdict
 from fluent.syntax import FluentParser, ast
 
-
```
Comment (mathjazz) [inline pontoon/checks/libraries/pontoon.py]: I would keep this blank line. As you can check in other files, we usually split imports and global variable definitions with two blank lines.

File: pontoon/checks/libraries/pontoon.py [hunk 2]
```diff
@@ -32,9 +34,19 @@ def run_checks(entity, string):
...
+        html_parser.unescape(
+            bleach.clean(
+                string,
+                strip=True,
+                tags=()
+            )
+        )
+    )
 
     # Prevent translations exceeding the given length limit
```
Comment (mathjazz) [inline pontoon/checks/libraries/pontoon.py]: This comment should indicate the entire max length check. So it should be moved above the following line:
```
max_length = get_max_length(entity.comment)
```

**Comment by jotes** [comment]
@mathjazz fixed

**Comment by mathjazz** [comment]
Thanks!

**Pull request merged**